src/pages/en/guides/data-fetching.md

```diff
@@ -5,7 +5,24 @@ description: Learn how to fetch remote data with Astro using the fetch API.
 i18nReady: true
 ---
 
-`.astro` files can fetch remote data at build time to help generate your pages.
+`.astro` files can fetch data from your local files, or remote data at build time to help generate your pages.
```

Comment:
This sentence feels a bit wordy to me, what do you think of rephrasing it this way? I omitted local *files* but since the example mentions that we are fetching from a local file I think readers aren't going to be confused.
```suggestion
`.astro` files can fetch local and remote data at build time to help generate your pages.
```